package.json

```diff
@@ -70,7 +70,7 @@
     "typescript": "^5.5.3"
   },
   "dependencies": {
-    "elliptic": "^6.5.5"
+    "@noble/curves": "^1.6.0"
```

Comment:
if you only need secp256k1, you should be able to use [@noble/secp256k1](https://www.npmjs.com/package/@noble/secp256k1/v/1.5.3) instead.